### ISSUE TITLE
[BUG]: Error during package import

### ISSUE THREAD

**Issue by f-scotto** [Unity-Technologies/UnityRenderStreaming#965]
### Package version

3.1.0-exp.8

### Environment

```markdown
* OS: Win 10
* Unity version: 2022.3.9f1
* Graphics API: 
* Browser: Chrome
```

### Steps To Reproduce

1: Select Window > Package Manager in the menu bar.
2: Check Package Manager window, Click + button and select Add package from git URL.
3: Input the com.unity.renderstreaming

### Current Behavior

Following error appear in console:

Library\PackageCache\com.unity.renderstreaming@3.1.0-exp.7\Runtime\Scripts\VideoStreamSender.cs(691,76): error CS1503: Argument 2: cannot convert from 'bool' to 'Unity.WebRTC.CopyTexture'

### Expected Behavior

4: The Render Streaming Wizard window is opened automatically after installing the package. Select Fix All.

**Comment by karasusan** [comment]
@f-scotto 
Are you trying to use the "dev" branch?

**Comment by f-scotto** [comment]
@karasusan no, I am following the installation steps described [HERE ](https://docs.unity3d.com/Packages/com.unity.renderstreaming@3.1/manual/tutorial.html)

**Comment by karasusan** [comment]
@f-scotto
We haven't released 3.1.0-exp.8 version yet.
Please wait our next release or use the latest version 3.1.0-exp.7.

**Comment by srbentostk** [comment]
> @f-scotto We haven't released 3.1.0-exp.8 version yet. Please wait our next release or use the latest version 3.1.0-exp.7.

this is happening at 3.1.0-exp.7
for me at least

**Comment by srbentostk** [comment]
> > @f-scotto We haven't released 3.1.0-exp.8 version yet. Please wait our next release or use the latest version 3.1.0-exp.7.
> 
> this is happening at 3.1.0-exp.7 for me at least

my problem was the version of WebRTC package
